expect/index.d.ts

```diff
@@ -1,72 +1,100 @@
-// Type definitions for Expect v1.13.4
+// Type definitions for Expect v1.20.2
 // Project: https://github.com/mjackson/expect
 // Definitions by: Justin Reidy <https://github.com/jmreidy/>
+// Definitions by: Risto Keravuori <https://github.com/merrywhether/>
```

Comment:
Write your name on the same line. `// Definitions by: Justin Reidy <https://github.com/jmreidy/>, Risto Keravuori <https://github.com/merrywhether/>`